Dockerfile

```diff
@@ -5,7 +5,7 @@
 FROM golang:alpine AS build-env
 
 # Set up dependencies
-ENV PACKAGES python3 curl make git libc-dev bash gcc linux-headers eudev-dev
+ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3
```

Comment:
Not a big expert of alpine's package manager here, but how does this affect the build of the image? It seems like that changing the order of the packages matters

Reply:
it makes it build. python is deprecated. pythong2 has officially been deprecated

Reply:
Yes, I know :) It just looked weird that the order of the packages mattered. Anyways, nevermind :+1:

Reply:
oh no this was my mistake. I accidentally pushed python3 before curl directly to rc1, so this commit moved it back to where it was before